cypress/e2e/APIRequests.cy.ts

```diff
@@ -0,0 +1,51 @@
+import { mixedResponse, emptyDiagnosisOptions, emptyAssessmentToolOptions } from '../fixtures/mocked-responses';
+
+describe('API request', () => {
+  it('Intercepts the request sent to the API and asserts over the request url', () => {
+    cy.intercept({
+      method: 'GET',
+      url: 'query/?*',
+    }, mixedResponse).as('call');
+    cy.intercept({
+      method: 'GET',
+      url: '/nodes/',
+    }).as('getNodes');
+    cy.visit('/?node=OpenNeuro');
+    // We need to wait for the fetch to complete and populate the
+    // dropdown with nodes and selecting OpenNeuro before making the request
+    cy.wait('@getNodes', { timeout: 10000 });
```

Comment:
very nice. But why do we need a timeout as well? Shouldn't this call be instantaneous and always succeed - because we're intercepting it?

Reply:
ah, I guess you don't actually stub the request here? Wouldn't that be a better option here? I.e. just git some nodes back for the test and avoid actually calling the API (which might be down etc)

Reply:
> why do we need a timeout as well? Shouldn't this call be instantaneous and always succeed - because we're intercepting it?

Left over from debugging

> ah, I guess you don't actually stub the request here? Wouldn't that be a better option here? I.e. just git some nodes back for the test and avoid actually calling the API (which might be down etc)

No, we don't but we could do that like we do in the test cases below